docs/content/deployment/guides/kubernetes/customizing-your-deployment.mdx

```diff
@@ -214,41 +214,77 @@ Other executors will ignore the `dagster-k8s/config` tag when it is set on an op
 
 ### Precedence rules
 
-By default, if a Kubernetes configuration dictionary (like `container_config`) is specified at both the instance level in the Helm chart and in a specific Dagster job or op, the dictionaries will be shallowly merged. The more specific configuration takes precedence if the same key is set in both dictionaries.
+By default, if Kubernetes configuration is specified at both the instance level in the Helm chart and in a specific Dagster job or op, the configuration is merged recursively. Scalar values will be replaced by the more specific configuration, dictionary fields will be combined, and list fields will be appended to each other, discarding duplicate values. List fields that cannot be meaningfully appended, like `command` or `args`, are replaced.
```

Comment:
`is specified at both the instance level in the helm chart and in specific Dagster job or op`. 

what about `dagster_cloud.yaml` code locations - I would expect it works the same. If so, any reason that we're omitting this method? are we intentionally directing users to configure solely through helm charts?